### PR TITLE
frontend: update cointracking link

### PR DESCRIPTION
**Pull request by jstrnbrg**
Cointracking asked us to append a unique ref to their link.

### CODE REVIEW

File: backend/backend.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ var fixedURLWhitelist = []string{
 	"https://bitcoincore.org/en/2016/01/26/segwit-benefits/",
 	"https://en.bitcoin.it/wiki/Bech32_adoption",
 	// Others
-	"https://cointracking.info/import/bitbox/",
+	"https://cointracking.info/import/bitbox/?aa=511",
```
Comment (benma) [inline backend/backend.go:89]: i don't think this is needed here, the whitelist is a prefix-check only

**Review by benma — CHANGES_REQUESTED** [review]
please change commit msg from `backend:` to `frontend:` too

**Comment by jstrnbrg** [comment]
All good? @benma

**Comment by benma** [comment]
@jstrnbrg did you run this change and click the link? did it work?

**Comment by jstrnbrg** [comment]
> @jstrnbrg did you run this change and click the link? did it work?

yes I did test it locally :)
Screenshot 2022-05-25 at 12 57 14

**Comment by benma** [comment]
@jstrnbrg including clicking it? :D

**Comment by jstrnbrg** [comment]
> @jstrnbrg including clicking it? :D

yes lol 
Screenshot 2022-05-25 at 13 11 43

**Review by benma — APPROVED** [review]
utACK

**Pull request merged**